### PR TITLE
hud: Add main toolbar button stats

### PR DESCRIPTION
**Pull request by kingthorin**
- Add stats counter to main toolbar button action.
- Add note to CHANGELOG.

Part of zaproxy/zaproxy#8375.

### CODE REVIEW

**Comment by kingthorin** [comment]
Test failure seems to be the "normal" Firefox one.

File: src/main/java/org/zaproxy/zap/extension/hud/ExtensionHUD.java [hunk 1]
```diff
@@ -368,6 +369,7 @@ private ZapToggleButton getHudButton() {
                         hudEnabledForDesktop = hudButton.isSelected();
                         getHudParam().setEnabledForDesktop(hudEnabledForDesktop);
                         setZapCanGetFocus(!hudEnabledForDesktop);
+                        Stats.incCounter("stats.ui.toolbar.button.hud");
```
Comment (thc202) [inline src/main/java/org/zaproxy/zap/extension/hud/ExtensionHUD.java:372]: This should be updated to match the others now.

Comment (kingthorin) [reply src/main/java/org/zaproxy/zap/extension/hud/ExtensionHUD.java:372]: Fixed

Comment (thc202) [reply src/main/java/org/zaproxy/zap/extension/hud/ExtensionHUD.java:372]: No change was pushed.

Comment (kingthorin) [reply src/main/java/org/zaproxy/zap/extension/hud/ExtensionHUD.java:372]: Ugh, not sure what happened here. Will check.

Comment (kingthorin) [reply src/main/java/org/zaproxy/zap/extension/hud/ExtensionHUD.java]: Fixed, for real 😉

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**